webui/react/src/services/api.ts

```diff
@@ -80,33 +81,58 @@ export const getExperimentDetails =
 export const getTrialDetails =
   generateApi<TrialDetailsParams, TrialDetails>(Config.getTrialDetails);
 
-export const killExperiment = generateDetApi<KillExpParams, Api.V1KillExperimentResponse, void>(
-  Config.killExperiment,
+export const forkExperiment = generateApi<ForkExperimentParams, number>(Config.forkExperiment);
+
+export const activateExperiment = generateDetApi<
+ExperimentIdParams,
+Api.V1ActivateExperimentResponse,
+void
+> (
+  Config.activateExperiment,
 );
 
-export const forkExperiment = generateApi<ForkExperimentParams, number>(Config.forkExperiment);
+export const archiveExperiment = generateDetApi<
+ExperimentIdParams,
+Api.V1ArchiveExperimentResponse,
+void
+> (
+  Config.archiveExperiment,
+);
 
-export const patchExperiment = generateApi<PatchExperimentParams, void>(Config.patchExperiment);
+export const cancelExperiment = generateDetApi<
+ExperimentIdParams,
+Api.V1CancelExperimentResponse,
+void
+> (
+  Config.cancelExperiment,
+);
 
-export const archiveExperiment = async (id: number, archive = true): Promise<void> => {
-  try {
-    await archive ?
-      Config.detApi.Experiments.determinedArchiveExperiment(id) :
-      Config.detApi.Experiments.determinedUnarchiveExperiment(id);
-  } catch (e) {
-    processApiError('archiveExperiment', e);
-    throw e;
-  }
-};
+export const killExperiment = generateDetApi<
+ExperimentIdParams,
+Api.V1KillExperimentResponse,
+void
+> (
+  Config.killExperiment,
+);
 
-export const setExperimentState = async (
-  { state, ...rest }: PatchExperimentState,
-): Promise<void> => {
-  return await patchExperiment({
-    body: { state },
-    ...rest,
-  });
-};
+export const pauseExperiment = generateDetApi<
+ExperimentIdParams,
+Api.V1PauseExperimentResponse,
+void
+> (
+  Config.pauseExperiment,
+);
+
+export const unarchiveExperiment = generateDetApi<
```

Comment:
nit: keep the pairs together? unarchive and archive, pause and activate, etc

Reply:
I'm not sure exactly what you mean?

Reply:
I think Hamid is suggesting to have this `unarchiveExperiment` function sit right below `archiveExperiment` function in `api.ts` because they are related. Same with pause/activate and maybe cancel/kill (different relationship but related).

Reply:
The order should make more sense now.

Reply:
it's great